lib/core/src/server/preview/to-require-context.js

```diff
@@ -1,14 +1,37 @@
 import globBase from 'glob-base';
-import { makeRe } from 'nanomatch';
+import { makeRe } from 'micromatch';
+
+// LEGACY support for bad glob patterns we had in SB 5 - remove in SB7
+const fixBadGlob = (val) => {
+  const match = val.match(/\.(\([^)]+\))/);
+
+  if (match) {
+    return match.input.replace(match[1], `@${match[1]}`);
```

Comment:
Can we add a `util-deprecate` deprecation warning here?

Reply:
good call, will add it